pkg/mountmanager/mount_linux.go

```diff
@@ -75,3 +104,55 @@ func (m *NodeMounter) Resize(devicePath string, deviceMountPath string) (bool, e
 	}
 	return true, nil
 }
+
+// createMountHelperContainerRequest creates a request to mount-helper-container server over UNIX socket and returns errors if any.
+func createMountHelperContainerRequest(payload string, url string) (string, error) {
+	// Get socket path
+	socketPath := os.Getenv("SOCKET_PATH")
+	if socketPath == "" {
+		socketPath = defaultSocketPath
+	}
+	// Create a custom dialer function for Unix socket connection
+	dialer := func(ctx context.Context, network, addr string) (net.Conn, error) {
+		return net.Dial("unix", socketPath)
+	}
+
+	// Create an HTTP client with the Unix socket transport
+	client := &http.Client{
+		Transport: &http.Transport{
+			DialContext: dialer,
+		},
+		Timeout: 3 * time.Minute, // Put 3 min timeout
```

Comment:
This 3 mins should be configurable or const not like this hard coded value

Reply:
Moved as a const